ntoskrnl/ob/obhandle.c

```diff
@@ -60,6 +60,11 @@ ObGetProcessHandleCount(IN PEPROCESS Process)
 
     ASSERT(Process);
 
+    /* Idle Process shares the HandleTable with PsInitialSystemProcess so let's return these
```

Comment:
Please move this to the ObGetProcessHandleCount call in the SystemProcessInformation handling. This is a hack needed only there.

Reply:
Done